cmd/network/evpn-utils.go

```diff
@@ -37,12 +37,17 @@ func ComposeGwIps(comp []*pc.IPPrefix) string {
 
 // PrintLB prints the logical bridge fields in human readable format
 func PrintLB(lb *pb.LogicalBridge) {
-	Vteip := fmt.Sprintf("%+v/%v", ipconv.IntToIPv4(lb.GetSpec().GetVtepIpPrefix().GetAddr().GetV4Addr()), lb.GetSpec().GetVtepIpPrefix().GetLen())
 	log.Println("name:", lb.GetName())
 	log.Println("status:", lb.GetStatus().GetOperStatus().String())
 	log.Println("vlan:", lb.GetSpec().GetVlanId())
-	log.Println("vni:", lb.GetSpec().GetVni())
-	log.Println("VtepIpPrefix:", Vteip)
+	if lb.GetSpec().GetVni() != 0 {
+		log.Println("vni:", lb.GetSpec().GetVni())
+	}
+	if lb.GetSpec().GetVtepIpPrefix().GetAddr().GetV4Addr() != 0 {
+		Vteip := fmt.Sprintf("%+v/%v", ipconv.IntToIPv4(lb.GetSpec().GetVtepIpPrefix().GetAddr().GetV4Addr()), lb.GetSpec().GetVtepIpPrefix().GetLen())
```

Comment:
why we cannot just print zeroes here?

Reply:
There are cases where we dont pass the vni or vtep ip i.e vrf without vni & vtep. Then we dont expect them to be printed as zeros.